### PR TITLE
Clippy and rustc

### PR DESCRIPTION
**Pull request by 4rmanc**

### CODE REVIEW

File: client/src/lib.rs [hunk 1]
```diff
@@ -21,27 +21,27 @@
...
-// 	absolute_paths_not_starting_with_crate, deprecated, future_incompatible, missing_docs,
-// 	nonstandard_style, unreachable_code, unreachable_patterns
-// )]
+#![deny(
+	absolute_paths_not_starting_with_crate, deprecated, future_incompatible, nonstandard_style,
+	unreachable_code, unreachable_patterns
+)]
 #![forbid(unsafe_code)]
 // Clippy
 // #![allow(clippy::tabs_in_doc_comments)]
```
Comment (brech1) [inline client/src/lib.rs:30]: We could remove this rule from the client, wdyt?

Comment (lazovicff) [reply client/src/lib.rs:30]: Sure, I don't see the need for keeping this atm

Comment (lazovicff) [reply client/src/lib.rs:30]: @4rmanc Can you remove this for now, so we can merge this?

Comment (lazovicff) [reply client/src/lib.rs:30]: @brech1 Are you talking about clippy::tabs_in_doc_comments?

Comment (brech1) [reply client/src/lib.rs:30]: Yes 👍🏼

**Review by lazovicff — DISMISSED** [review]

**Review by brech1 — APPROVED** [review]

**Pull request merged**